unrasterize/classes.py

```diff
@@ -146,12 +146,13 @@ def _reassign_pixel_values(band, pixels, raw_pixel_values=[]):
         """
         if not raw_pixel_values:
             raw_pixel_values = [band[tuple(pixel)] for pixel in pixels]
-        # Avoid underflow by ignoring negative values.
-        total = np.sum(band[band > 0.0], dtype=np.float32)
+        # Avoid underflow by ignoring values below the threshold.
+        total = np.sum(band[band > threshold], dtype=np.float32)
```

Comment:
I'm not convinced that passing `threshold=self.threshold` works. 

[The example notebook](https://github.com/tetraptych/unrasterize/blob/master/examples/basic_unrasterizer_usage.ipynb) uses an `Unrasterizer(mask_width=10, threshold=0.2)` object. This function would then ignore all pixels with a population density < 0.2, so that they would not contribute to the total that is distributed among the selected pixels. In other words, the total sum of the selected pixels' values after applying this function would not equal the total sum across all pixels.